src/deploy/osp_deployer/director.py

```diff
@@ -1057,6 +1067,49 @@ def setup_powermax_cinder(self, powermax_cinder_yaml):
         for cmd in cmds:
             self.run_tty(cmd)
 
+    def setup_powermax_manila(self, powermax_manila_yaml):
+
+        if self.settings.enable_powermax_manila_backend is False:
+            logger.debug("Not setting up powermax manila backend.")
+            return
+
+        logger.debug("Configuring dell emc powermax manila backend.")
+
+        overcloud_images_file = self.home_dir + "/overcloud_images.yaml"
+
+        cmds = ['sed -i "s|<manila_powermax_driver_handles_share_servers>|' +
+                self.settings.manila_powermax_driver_handles_share_servers +
+                '|" ' + powermax_manila_yaml,
+                'sed -i "s|<manila_powermax_nas_login>|' +
+                self.settings.manila_powermax_nas_login + '|" ' +
+                powermax_manila_yaml,
+                'sed -i "s|<manila_powermax_nas_password>|' +
+                self.settings.manila_powermax_nas_password + '|" ' +
+                powermax_manila_yaml,
+                'sed -i "s|<manila_powermax_nas_server>|' +
+                self.settings.manila_powermax_nas_server + '|" ' +
+                powermax_manila_yaml,
+                'sed -i "s|<manila_powermax_server_container>|' +
```

Comment:
Looks like a typo here , should be manila_powermax_server_container**s**

Reply:
It is the name of the data mover. it is server_container

Reply:
it s container**b** with a "s" in the yaml file ? so the operation without the s at the end above wont sed properly

Reply:
You are right, I fixed the yaml file